x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/JdbcTestUtils.java

```diff
@@ -20,6 +23,8 @@
     public static final String SQL_TRACE = "org.elasticsearch.xpack.sql:TRACE";
 
     public static final String JDBC_TIMEZONE = "timezone";
+    
+    public static ZoneId UTC = ZoneId.of("Z");
```

Comment:
Any reason why `org.es.xpack.sql.util.DateUtils` is not used instead?

Reply:
It is not visible in the `jdbc` project.